pages/impact.js

```diff
@@ -0,0 +1,78 @@
+import { NextSeo } from 'next-seo';
+import Image from 'next/image';
+import React from 'react';
+
+import Banner from '../components/Banner';
+import Carousel from '../components/Impact/Carousel';
+import PersonCard from '../components/Impact/PersonCard';
+import WorkshopCard from '../components/Impact/WorkshopCard';
+import Layout from '../components/Layout';
+
+import { pastEvents, people } from '../data/impact';
+
+import impactMotifBanner from '../public/images/impact/impact-motif.png';
+import styles from '../styles/pages/impact.module.scss';
+
+const impactBlog = 'https://medium.com/acm-at-ucla';
+
+function Impact() {
+	return (
+		<Layout>
+			<NextSeo
+				title="ACM Impact | ACM at UCLA"
+				description="ACM Impact is an up-and-coming initiative of ACM whose mission is to start conversations and promote an understanding of how tech and society affect each other. Our aim is to build a community of people who are passionate about ethical technology, socially impactful engineering, and creating a better future for all!"
+				openGraph={{
+					images: [
+						{
+							url: 'https://www.uclaacm.com/images/impact/impact-motif.png',
+							width: 2665,
+							height: 1333,
+							alt: 'The ACM Impact banner: a space shuttle taking off, with a moon behind it',
+						},
+					],
+					site_name: 'ACM at UCLA',
+				}}
+			/>
+			<Banner decorative />
+			<div className={styles.container}>
+				<div className="text-center">
+					<Image src={impactMotifBanner} alt='Impact by ACM at UCLA' />
+				</div>
+
+				<h2 className="text-center">Tech Ethics.</h2>
+				{/* TODO: make this not use an h2 */}
+				<h2 className={styles['text-center-holographic']}>Ethics.</h2>
```

Comment:
Curious why the preference is to have this not be an `<h2>`

Reply:
So the problem is that this is a decorative tag, it duplicates the text from above. Would like to restyle to an aria-hidden `<span>`, but I think that breaks the styling right now; need to investigate further.